marathon.json

```diff
@@ -0,0 +1,13 @@
+{
+    "id": "hdfs",
+    "container": {
```

Comment:
for now i'm going with the URI strategy to prevent confusion about what the hdfs scheduler is actually doing. Full docker support is upcoming.